clang/test/AST/ByteCode/constexpr-vectors.cpp

```diff
@@ -0,0 +1,90 @@
+// RUN: %clang_cc1 %s -triple x86_64-linux-gnu -Wno-uninitialized -std=c++14 -fsyntax-only -verify
+// RUN: %clang_cc1 %s -triple x86_64-linux-gnu -fexperimental-new-constant-interpreter -Wno-uninitialized -std=c++14 -fsyntax-only -verify
```

Comment:
Is the `-Wno-uninitialized` needed here?

Reply:
It has no effect in the current test, just to keep it the same as test/SemaCXX/constexpr-vectors.cpp.

Reply:
Removed.